nuancier/static/nuancier.css

```diff
@@ -5,7 +5,10 @@
 .error, .required {
     color: red;
 }
-
+.hyphen{
+    margin-left:80px;
+    margin-right:80px;
+}
```

Comment:
Small nitpicking, but I like to keep a blank line between my css declarations, so one before and one after :-)